### PR TITLE
Added Fetch TV renderer

### PR DESCRIPTION
**Pull request by jasminsehic**
https://www.fetchtv.com.au/

### CODE REVIEW

File: src/main/external-resources/renderers/FetchTV.conf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
+# ============================================================================
+#
+
+UserAgentSearch = Linux/3.14.28-1.6.*DLNADOC/1.50
+
+TranscodeVideo = MPEGTS-H264-AC3
+TranscodeAudio = MP3
+MuxLPCMToMpeg = false
```
Comment (Nadahar) [inline src/main/external-resources/renderers/FetchTV.conf]: Thank you for your contribution. There's a couple of things that's hard for an outsider to know. ```MuxLPCMToMpeg``` isn't used if ```MediaInfo = true```, but the ```Supported``` lines you've defined isn't used unless ```MediaInfo = true```.

Parsing without MediaInfo is the old, crude way of parsing with UMS, and in most cases it won't be very accurate. You should enable MediaInfo, so I'd replace this line with ```MediaInfo = true```.

**Comment by Nadahar** [comment]
@jasminsehic Great. Have you tested once MediaInfo was enabled?

File: src/main/external-resources/renderers/FetchTV.conf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+RendererIcon = fetch-tv.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
+# ============================================================================
+#
+
+UserAgentSearch = Linux/3.14.28-1.6.*DLNADOC/1.50
```
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Not renderer specific, should be changed.

**Comment by jasminsehic** [comment]
@Nadahar yes I watched a bunch of videos to make sure. Looks good.

**Comment by Sami32** [comment]
@jasminsehic Thank you for sharing your nice work :+1:

File: src/main/external-resources/renderers/FetchTV.conf (hunk 2 above)
Comment (jasminsehic) [inline src/main/external-resources/renderers/FetchTV.conf]: @Sami32 thanks for the feedback. Renderer does not seem to uniquely identify itself so I am not sure what to change it to. Any suggestions?

Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: I cannot without your full trace log ;-)
https://github.com/UniversalMediaServer/UniversalMediaServer/blob/4307d45adac8b8172e07f0e4971405a2cf9be630/CONTRIBUTING.md

Comment (jasminsehic) [inline src/main/external-resources/renderers/FetchTV.conf]: @Sami32 here it is. Thanks.
[ums_dbg.zip](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1203223/ums_dbg.zip)

File: src/main/external-resources/renderers/FetchTV.conf [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Fetch TV
+RendererIcon = fetch-tv.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
```
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Should be replaced by this:
```
# User-Agent: Takin/3.0.0 (Linux arm ; U; en), FetchTV_STB_BCM7252S/3.7.7244 (FetchTV, M616T, Wireless)
# USER-AGENT: Linux/3.14.28-1.6, HTTP/1.0, Portable SDK for HTTP devices
# uuid:6668fc05-b140-4154-bf17-7261d399cb82
```

File: src/main/external-resources/renderers/FetchTV.conf (hunk 2 above)
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Thanks.
I think you should use that one instead:
`User-Agent: Takin/3.0.0 (Linux arm ; U; en), FetchTV_STB`

**Comment by jasminsehic** [comment]
@Sami32 great thank you! I've updated it all now.

**Comment by Sami32** [comment]
I guess that you already saw that you've forgotten to rename the thumbnail file:
`src/main/resources/images/clients/fetch-tv.png`

It need to be same as `RendererIcon = Fetch-TV.png` or it will not work.

**Comment by Sami32** [comment]
After that, you will need to try it and test if it this renderer is detected correctly, and that M4A, AIFF and AAC files play as expected.

File: src/main/external-resources/renderers/FetchTV.conf [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+UserAgentSearch = FetchTV
+
+TranscodeVideo = MPEGTS-H264-AC3
+TranscodeAudio = MP3
+TranscodeFastStart = true
+MediaInfo = true
+
+# Supported video formats (see https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf)
+Supported = f:avi       v:xvid                                       a:mp3                                         m:video/avi
```
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Better to delete `(see https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf)`

File: src/main/external-resources/renderers/FetchTV.conf [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Supported = f:flv       v:sor                                        a:mp3                                         m:video/x-flv
+Supported = f:mkv       v:divx|h264|mp4|xvid                         a:ac3|mp3                                     m:video/x-matroska
+Supported = f:mov       v:mp4                                        a:aac-lc                                      m:video/quicktime
+Supported = f:mp4       v:mp4                                        a:aac-lc                                      m:video/mp4
+Supported = f:mpegts    v:h264|mpeg1|mpeg2|mp4                       a:aac-lc|ac3|mp3|mp2                          m:video/vnd.dlna.mpeg-tts
+Supported = f:vob       v:mpeg2                                      a:mp2                                         m:video/x-ms-vob
+Supported = f:wmv       v:wmv                                        a:wma                                         m:video/x-ms-wmv
+
+# Supported audio formats (see https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf)
+Supported = f:adts      a:aac-lc|he-aac                     m:audio/aac
```
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Same as above

File: src/main/external-resources/renderers/FetchTV.conf [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+RendererName = Fetch TV
+RendererIcon = Fetch-TV.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Takin/3.0.0 (Linux arm ; U; en), FetchTV_STB_BCM7252S/3.7.7244 (FetchTV, M616T, Wireless)
+# USER-AGENT: Linux/3.14.28-1.6, HTTP/1.0, Portable SDK for HTTP devices
+# ============================================================================
+#
```
Comment (Sami32) [inline src/main/external-resources/renderers/FetchTV.conf]: Here add: `# https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf`

**Comment by Nadahar** [comment]
@Sami32 Maybe it gets a bit too much for @jasminsehic. You can create a pull request against his branch in his fork where you fix the things you see, maybe that's easier? He can then merge it if he wants to and this PR will be updated.

**Comment by jasminsehic** [comment]
@Sami32 the rendered detected OK for me since I am on Windows but I have changed it now. I will test out  M4A, AIFF and AAC files later today.

**Comment by Sami32** [comment]
@jasminsehic Great :+1: 
Thanks, take you time, nothing hurry ;-)

PLEASE, before merge it, select "**Squash and merge**"

**Comment by Sami32** [comment]
@Nadahar Yes, i was afraid to discourage him, but i disliked interfere in his PR without his consent.
As nothing hurry, i preferred wait before doing anything, and happily he made a very good work and didn't get discouraged, frustrated, and that's great.
I know that you can understand me :wink

**Comment by Nadahar** [comment]
@Sami32 What was the reason not to add the ```@2x``` image?

**Comment by Sami32** [comment]
@Nadahar The only one image that i found that could be able to satisfy our guideline renderers thumbnail, after image retouching, didn't have a suffisant resolution.

Since this renderer have a very basic thumbnail, upscale will give nothing more than an ugly thumbnail, that will be worst than not setting a 2x one, IMO.

**Comment by Nadahar** [comment]
@Sami32 Ok, I just didn't understand why it was dropped. I think I read that the upscaling done by macOS is *very* bad and that upscaling the image "manually" will give better quality, but I'm not sure. I do agree with you though, I'm against upscaling "on principle".

**Comment by Sami32** [comment]
@nadahar Why a such hurry ? :-(
He was on the way to test and merge it himself later.

**Comment by Nadahar** [comment]
@Sami32 He can't merge himself, only we can do that. Both you and @valib had approved it, which is supposed to mean that it's tested and ready. I did it because I wanted to make sure that it was squashed when merged, instead of some of you forgetting about that and merging 5 commits for one renderer.

**Comment by Sami32** [comment]
No, he told he will tes the audio later do it later.
I was afraid for the squash as well, it why i've boldly advertised.

**Comment by Nadahar** [comment]
It would be better if you had waited with the approval then, that was what made me think it was ready. There's not much we can do about it now anyway, so let's just hope that the audio is correct. If not, you can correct it in #1320.

**Comment by Sami32** [comment]
:stuck_out_tongue:

**Comment by jasminsehic** [comment]
@Nadahar & @Sami32 M4A, AIFF and AAC audio files played OK.

**Pull request merged**